### PR TITLE
Allow conditional compilation for app extension targets

### PR DESCRIPTION
**Pull request by lucianoiam**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -16,3 +16,7 @@ xcuserdata
 DerivedData
 *.xcuserstate
 node_modules/
+
+### macOS Finder ###
```
Comment (skywinder) [inline .gitignore:20]: this is should be in local gitignore file

Comment (lucianoiam) [reply .gitignore]: left .gitignore unchanged and force-pushed

**Comment by lucianoiam** [comment]
All merge conflicts have been solved

**Comment by lucianoiam** [comment]
@noorulain17 any chance for this to be merged? it is needed for including ActionSheetPicker in [app extensions](https://developer.apple.com/app-extensions/) like [AUv3](https://developer.apple.com/documentation/audiotoolbox/audio_unit_v3_plug-ins/creating_custom_audio_effects?language=objc). Thanks for reviewing.

**Comment by noorulain17** [comment]
Sure @lucianoiam.

Please squash the two commits and I'll merge.

Thanks for the contribution 🙂

**Comment by lucianoiam** [comment]
done!

**Comment by lucianoiam** [comment]
@noorulain17 any outstanding task on my side?. Sorry to re-ask but my first attempt last year (above) was unsuccessful so I feel I need to insist :) . Thanks.

**Comment by noorulain17** [comment]
Thanks @lucianoiam for the quick squash.

I'll just test the branch once and then will merge and release a new version.

**Pull request closed without merging**